src/javascript/app/pages/user/account/settings/account_closure.js

```diff
@@ -0,0 +1,139 @@
+const getCurrencyFullName = require('../../../../common/currency').getCurrencyFullName;
+const Client              = require('../../../../base/client');
+const BinarySocket        = require('../../../../base/socket');
+const localize            = require('../../../../../_common/localize').localize;
+const Url                 = require('../../../../../_common/url');
+const isCryptocurrency    = require('../../../../../_common/base/currency_base').isCryptocurrency;
+
+const AccountClosure = (() => {
+    const form_selector = '#form_closure';
+    let $form,
+        $txt_other_reason,
+        $closure_loading,
+        $closure_container,
+        $success_msg,
+        $error_msg,
+        $crypto,
+        $fiat;
+
+    const onLoad = () => {
+        $txt_other_reason  = $('#other_reason');
+        $closure_loading   = $('#closure_loading');
+        $closure_container = $('#closure_container');
+        $success_msg       = $('#msg_main');
+        $error_msg         = $('#msg_form');
+        $crypto            = $('#crypto');
+        $fiat              = $('#change-fiat');
+        $form              = $(form_selector);
+
+        BinarySocket.wait('landing_company').then((response) => {
+            const company = response.landing_company.gaming_company || response.landing_company.financial_company;
```

Comment:
should we add handler to check if the response is exists?

Reply:
What's a good way to handle that?

Reply:
better use getPropertyValue()

Reply:
you shouldn't fallback, check if you should use gaming or financial depending on what the current landing company of the client is (landing_company_shortcode should match)